foursight_core/react/api/envs.py

```diff
@@ -85,12 +87,9 @@ def get_user_auth_info(self, email: str) -> Tuple[list, list, str]:
                     # informational/display purposes in the UI.
                     first_name = user.get("first_name")
                     last_name = user.get("last_name")
-                    if True:
-                        # New; 2022-10-25; almost forgot; from app_utils.check_authorization; check groups.
-                        groups = user.get("groups")
-                        if groups and "admin" in groups or "foursight" in groups:
-                            allowed_envs.append(known_env["full_name"])
-                    else:
+                    # New; 2022-10-25; almost forgot; from app_utils.check_authorization; check groups.
+                    groups = user.get("groups")
+                    if groups and "admin" in groups or "foursight" in groups:
```

Comment:
Can we abstract this into a permissions tester rather than doing something so low-level amid a function doing something different? e.g.,
```
if in_one_or_more_groups(user, ["admin", "foursight"]):
    ...
```
and then define (untested):
```
def in_one_or_more_groups(user, allowed_groups):
    user_groups = user.get("groups")
    return user_groups and any(allowed_group in user_groups for allowed_group in allowed_groups)
```

Reply:
Yes, good idea, thanks.